pkg/build/api/v1/conversion.go

```diff
@@ -91,6 +91,13 @@ func Convert_v1_BuildTriggerPolicy_To_api_BuildTriggerPolicy(in *BuildTriggerPol
 	switch in.Type {
 	case ImageChangeBuildTriggerTypeDeprecated:
 		out.Type = newer.ImageChangeBuildTriggerType
+		if in.ImageChange == nil && out.ImageChange == nil {
+			out.ImageChange = &newer.ImageChangeTrigger{}
+		}
+	case ImageChangeBuildTriggerType:
+		if in.ImageChange == nil && out.ImageChange == nil {
+			out.ImageChange = &newer.ImageChangeTrigger{}
+		}
```

Comment:
hm. we don't generally assign/initialize as part of conversion.  why isn't the defaulter getting called?